test_psa_target.py

```diff
@@ -284,20 +332,6 @@ def _get_parser():
         choices=["ARMCLANG", "GNUARM"],
     )
 
-    parser.add_argument(
-        "-d",
-        "--disk",
-        help="Target disk (mount point) path",
-        default=None,
-    )
-
-    parser.add_argument(
-        "-p",
-        "--port",
-        help="Target port for connection",
-        default=None,
-    )
```

Comment:
This is removing the feature of running test suites on a target which is connected to a host machine, which normally aids the development of new target being added. Are we getting rid of this feature?

Reply:
mbedgt will run tests against local device. It checks what is connected and runs tests specified in test_spec.json